README.md

```diff
@@ -9,16 +9,26 @@ Provides a platform for recording, aggregating, publishing and accessing metrics
 
 ###Versions
 
-#### 2G *(Proposed)*
+#### 2H *(Proposed)*
 
 * *Example Not Available*
 * *Schema Not Available*
 
 Changes:
 * Include time offset relative to initTimestamp on each timer sample.
-* Binary file format in addition to standard and steno wrapped JSON.
 * Network sink for writing directly to Tsd Aggregator using TCP or UDP.
 
+#### 2G
+
+** [Example](examples/query-log-example-2g.json)
+** [Schema](schema/query-log-schema-2g.json)
+
+Changes:
+* Add Support for dimensions
+* Remove restriction against additional root fields
+* Simplify annotations by extracting id and date
+* [Binary file format](https://github.com/InscopeMetrics/client-protocol) in addition to standard and steno wrapped JSON.
```

Comment:
Unless something changed from the other thread there will be no steno wrapped version of 2g; correct?